.github/workflows/checks.yml

```diff
@@ -24,6 +24,8 @@ on:
       - checks-workflow-dev/*
     tags:
       - 'v[0-9]+.[0-9]+.[0-9]+*'
+    paths-ignore:
+      - 'website/**'
```

Comment:
I think we should also ignore the `docs` folder

Reply:
Agreed, will append it to the list.